pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/MultiNodesOfflineClusterIntegrationTest.java

```diff
@@ -178,11 +178,12 @@ private void testCountStarQuery(int expectedNumServersQueried, boolean exception
       JsonNode exceptions = queryResult.get("exceptions");
       assertEquals(exceptions.size(), 2);
       JsonNode firstException = exceptions.get(0);
+      // NOTE:
+      // Only verify the error code but not the exception message because there can be different messages:
+      // - Connection refused
+      // - Connection reset
+      // - Channel is inactive
       assertEquals(firstException.get("errorCode").intValue(), QueryException.BROKER_REQUEST_SEND_ERROR_CODE);
-      String firstExceptionMessage = firstException.get("message").textValue();
-      assertTrue(
-          firstExceptionMessage.contains("Connection refused") || firstExceptionMessage.contains("Connection reset"),
-          "Got unexpected first exception message: " + firstExceptionMessage);
```

Comment:
You are not adding `|| firstExceptionMessage.contains("inactive")` here, is it because there might be other exception messages?

Reply:
The message itself is not that important, and is from a third-party library, so ignoring the actual content should be fine